openmdao/core/system.py

```diff
@@ -306,7 +306,7 @@ def _set_vars_as_remote(self):
             meta['remote'] = True
 
     def fd_jacobian(self, params, unknowns, resids, total_derivs=False,
-                    fd_params=None, fd_unknowns=None,
+                    fd_params=None, fd_unknowns=None, pass_unknowns=None,
```

Comment:
I would suggest using a default value of () for pass_unknowns instead of None. That way, you don't have to check later for a value of None.